config/environments/development.rb

```diff
@@ -90,13 +89,13 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.default_url_options = { host: config.app_domain }
   config.action_mailer.smtp_settings = {
-    address: "smtp.gmail.com",
-    port: "587",
-    enable_starttls_auto: true,
-    user_name: '<%= ENV["DEVELOPMENT_EMAIL_USERNAME"] %>',
-    password: '<%= ENV["DEVELOPMENT_EMAIL_PASSWORD"] %>',
-    authentication: :plain,
-    domain: config.app_domain
+    address: ENV["SMTP_ADDRESS"],
+    port: ENV["SMTP_ADDRESS"],
+    authentication: ENV["SMTP_AUTHENTICATION"],
+    user_name: ENV["SMTP_USER_NAME"],
+    password: ENV["SMTP_PASSWORD"],
+    domain: ENV["SMTP_DOMAIN"] || config.app_domain,
```

Comment:
Since Ruby considers empty string true, this might never fall back:

```suggestion
    domain: ENV["SMTP_DOMAIN"].presence || config.app_domain,
```